libs/components/lists/src/lib/modules/repeater/repeater.component.spec.ts

```diff
@@ -1212,6 +1212,19 @@ describe('Repeater item component', () => {
 
       flushDropdownTimer();
     }));
+
+    it('should be accessible', async () => {
+      cmp.showRepeaterWithActiveIndex = true;
+      cmp.expandMode = 'none';
+      cmp.activeIndex = 0;
+      // Detect active index.
```

Comment:
Put some empty lines between these logical blocks for readability.

```
it('should be accessible', async () => {
      cmp.showRepeaterWithActiveIndex = true;
      cmp.expandMode = 'none';
      cmp.activeIndex = 0;

      // Detect active index.
      fixture.detectChanges();
      await fixture.whenStable();

      // Role changes on next cycle.
      fixture.detectChanges();
      await fixture.whenStable();

      await expectAsync(fixture.nativeElement).toBeAccessible();
```

Reply:
updated